scripts/gen-tx-gov.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+if test "$#" -ne 2
+then
+   echo "error: 2 parameters are expected (patch version and upgrade height number)"
+   echo "example : ./gen-tx-gov.sh v0.16.1 1336350"
+    exit
+fi
+
+if [[ ! "$1" =~ v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3} ]]; 
+then 
+   echo "error: a semantic tag is expected as first parameter"
+   echo  "example v0.16.1"
+   exit
+fi
+
+re='^[0-9]+$'
+if ! [[ "$2" =~ $re ]]
+then
+   echo "error: an integer is expected as second parameter" >&2
+   exit
+fi
+
+PATCH="$1"
+MINOR=$(echo $PATCH | sed 's/..$//g')
+HEIGHT_NUMBER="$2"
+
+# LINUX AMD64
+curl --silent -LJO "https://github.com/axelarnetwork/axelar-core/releases/download/${PATCH}/axelard-linux-amd64-${PATCH}.zip.sha256"
```

Comment:
Perhaps create a bash function that takes two arguments, OS and ARCH and does this step to reduce code duplication, and then call it 4 times with os/arch combinations?

Reply:
indeed, we can reduce code duplication